app/controllers/catalog_controller.rb

```diff
@@ -150,6 +150,17 @@ def catalog_item_types
     }
   end
 
+  def generic_item_subtypes
+    {
+      "custom"        => "Custom",
+      "vm"            => "VM",
+      "instance"      => "Instance",
```

Comment:
I have never liked the cloud "instance" name as it is confusing out of context.  Maybe these values should be vm_infra, vm_cloud.

@dclarizio @chriskacerguis Thoughts?